parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -744,14 +746,17 @@ loop = do
           headb <- importRemoteBranch headRepo SyncMode.ShortCircuit
           lift $ do
             mergedb <- eval . Eval $ Branch.merge baseb headb
+            squashedb <- eval . Eval $ Branch.merge' Branch.SquashMerge headb baseb
```

Comment:
Yes, it'll be doing both on `pr.load`. With the namespace caching we do, merges have been pretty fast and it's just the code syncing that dominates the runtime - the code is only sync'd once and that actually populates the cache. It could impact runtime if the LCA is really far back and doesn't fit in the cache that far.

Reply:
If you change this line to `True` you'll get timing info for the various phases of `pr.load` and other commands: 

https://github.com/unisonweb/unison/blob/master/parser-typechecker/src/Unison/Util/Timing.hs#L13

Not a bad thing to just leave on for development.